src/main/java/com/ait/lienzo/client/core/shape/wires/WiresShape.java

```diff
@@ -1,24 +1,28 @@
 /*
-   Copyright (c) 2017 Ahome' Innovation Technologies. All rights reserved.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
```

Comment:
not sure if we should update the existing license...

Reply:
sure, you are right, the IDE automatically updated it =/